OpenRA.Utility/Program.cs

```diff
@@ -15,6 +15,16 @@
 
 namespace OpenRA.Utility
 {
+	public class NoSuchCommandException : ArgumentException
```

Comment:
How is `NoSuchCommandException` not an `ArgumentException` when it is about an argument.